### PR TITLE
Switch to carrierwave for for upload management

### PR DESCRIPTION
**Pull request by hennevogel**
It's way more flexible and brings simple support to use cloud storage like AWS or cloudinary. Additionally it fits perfect for our needs to host on PaaS services like heroku.

### CODE REVIEW

File: config/cloudinary.yml.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+production:
+  cloud_name: "sample"
+  api_key: "874837483274837"
```
Comment (differentreality) [inline config/cloudinary.yml.example]: Are those keys meant to be public?

Comment (hennevogel) [inline config/cloudinary.yml.example]: They are examples (taken from the cloudinary docu)

**Comment by differentreality** [comment]
Photos for conference, sponsor and logding look good! :+1: 

I don't think we are showing a photo for Venue anymore.. Not sure if that was intended in https://github.com/openSUSE/osem/pull/946

**Comment by hennevogel** [comment]
Yeah I've seen this. That Venue doesn't degrade anymore is a bug (#974) :-)

**Comment by hennevogel** [comment]
@differentreality so this can be merged?

**Comment by differentreality** [comment]
Yeap, let's do this :+1:

**Comment by nishanthvijayan** [comment]
@hennevogel  Now that they are not being used shouldn't we remove fields like logo_file_size, logo_content_type, logo_updated_at from the tables conferences sponsors and events?

**Pull request merged**